src/android/com/megster/cordova/BluetoothSerialService.kt

```diff
@@ -371,13 +373,20 @@ object BluetoothSerialService {
             } catch (e: IOException) {
                 Log.e(LOG_TAG, "temp sockets not created", e)
             }
+            try {
+                var bluetoothDevice: BluetoothDevice? = mmSocket?.getRemoteDevice()
+                Log.d(LOG_TAG, "Remote device's mac address: ${bluetoothDevice?.getAddress()}")
+                remoteDeviceMacAddress = bluetoothDevice?.getAddress()
```

Comment:
similar to typescript I'm pretty sure you can do:
```
if (bluetoothDevice) {
  Log.d(LOG_TAG, "Remote device's mac address: ${bluetoothDevice.getAddress()}")
  remoteDeviceMacAddress = bluetoothDevice.getAddress()
}
```
or if you want full swag points:
```
bluetoothDevice?.let {
  Log.d(LOG_TAG, "Remote device's mac address: ${it.address}")
  remoteDeviceMacAddress = it.address
```

Notice the switch from `getAddress` to `address`. The Kotlin compiler will replace `address` with `getAddress` at compile time, because it is idiomatic Kotlin to use properties instead of getter functions.